websites/L/Line TV/dist/metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "SlowLife",
+      "id": "374905512661221377"
+    },
+    "url": "tv.line.me",
+    "description": {
+      "en": "LINE TV is a global video streaming service. You can watch high-quality video content in a variety of categories such as TV dramas, entertainment shows."
+    },
+    "service": "Line TV",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/F2sWIqN.png",
+    "thumbnail": "https://i.imgur.com/5sqClBZ.png",
+    "color": "#ffffff",
+    "tags": [
```

Comment:
shouldnt you use the green color from their logo?

Reply:
Sure